Specs/Scene/ShadowMapSpec.js

```diff
@@ -362,30 +379,51 @@ defineSuite([
 
         // Render without shadows
         scene.shadowMap.enabled = false;
-        var unshadowedColor = render();
-        expect(unshadowedColor).not.toEqual(backgroundColor);
+        var unshadowedColor;
+        renderAndCall(function(rgba) {
+            unshadowedColor = rgba;
+            expect(unshadowedColor).not.toEqual(backgroundColor);
+        });
 
         // Render with shadows
         scene.shadowMap.enabled = true;
-        var shadowedColor = render();
-        expect(shadowedColor).not.toEqual(backgroundColor);
-        expect(shadowedColor).not.toEqual(unshadowedColor);
+        var shadowedColor;
+        renderAndCall(function(rgba) {
+            shadowedColor = rgba;
+            expect(rgba).not.toEqual(backgroundColor);
+            expect(rgba).not.toEqual(unshadowedColor);
+        });
 
         // Turn shadow casting off/on
         caster.shadows = ShadowMode.DISABLED;
-        expect(render()).toEqual(unshadowedColor);
+        expect({
+            scene : scene,
+            primeShadowMap : true
+        }).toRender(unshadowedColor);
```

Comment:
Could these call `renderAndReadPixels` instead? Or maybe call a similar function that calls just `toRender`? That way `primeShadowMap` only needs to appear in one place.